style.css

```diff
@@ -0,0 +1,740 @@
+  letter-spacing: -0.01em;
+  text-align: left;
+}
+
+/* footer */
+
+/* about */
+
+/* headline */
+.about-headline {
+  background-image: url("./images/about-headline.jpg");
+}
+
+.about-headline-introduction *,
+.about-headline-description {
+  text-align: center;
+}
+
+.about-headline-description {
+  font-family: Lato, sans-serif;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 25px;
+  text-align: center;
+}
+
+.about-headline-place {
+  font-family: Lato, sans-serif;
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 25px;
+  letter-spacing: 0.03em;
+  text-align: center;
+  color: #7a7a7a;
+}
+
+.about-headline-date {
+  font-family: Lato, sans-serif;
+  font-size: 17px;
+  font-weight: 800;
+  line-height: 25px;
+  letter-spacing: 0.02em;
+  text-align: center;
+  color: #5f5f5f;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.about-headline-date-place strong a {
+  color: unset;
+}
+
+.about-headline-date-place {
+  background-color: rgb(255, 255, 255, 0.7);
+}
+
+.about-headline .content-container {
+  max-width: 793px;
+}
+
+/* headline */
+
+/* alympic logo */
+.olympic-logo-title {
+  color: #000;
+}
+
+.olympic-logo .content-container {
+  max-width: 793px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 65px 0;
+}
+
+.logo-description {
+  font-family: Lato, sans-serif;
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 21px;
+  letter-spacing: -0.03em;
+  text-align: center;
+  color: #838383;
+  padding: 37px 0;
+}
+
+.logo-descption-image-container {
+  max-width: 200px;
+}
+
+.logo-description-image {
+  width: 100%;
+}
+
+/* alympic logo */
+
+/* past olympic */
+.past-olympic1,
+.past-olympic2 {
+  max-width: 432px;
+  min-width: 334px;
+  max-height: 236px;
+  min-height: 183.31px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.past-olympic1 {
+  background-image:
+    linear-gradient(
+      to bottom,
+      rgb(255, 79, 56, 0.4),
+      rgb(255, 79, 56, 0.4)
+    ),
+    url("images/beijing.webp");
+  background-size: cover;
+  margin: 0 0 25px 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.past-olympic2 {
+  background-image:
+    linear-gradient(
+      to bottom,
+      rgb(255, 79, 56, 0.4),
+      rgb(255, 79, 56, 0.4)
+    ),
+    url("images/tokyo.webp");
+  background-size: cover;
+  background-color: #ff4f38;
+}
+
+.past-olympic h3 {
+  font-family: Lato, sans-serif;
+  font-size: 30px;
+  font-weight: 800;
+  line-height: 21px;
+  letter-spacing: -0.03em;
+  text-align: center;
+  color: white;
+}
+
+.past-olympic p {
+  font-family: Lato, sans-serif;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 21px;
+  letter-spacing: -0.03em;
+  text-align: center;
+  color: white;
+  margin-top: 10px;
+}
+
+/* past olympic */
+
+/* about */
+
+@media screen and (min-width: 768px) {
+  /* header */
+  .top-nav {
```

Comment:
Hello @TadesseDev 
I used the mobile-first approach in this section because all the code I wrote above the media queries is for mobile, and the desktop one starts from 768px screens to bigger (use of min-width).
Please tell me what you mean by that.